src/templates/supported.html

```diff
@@ -8,19 +8,22 @@
     <h1>CubingUSA Supported Competitions</h1>
     <div>
       <p>
-        CubingUSA offers local organizers the option of running their competition under the auspices of CubingUSA.  Because CubingUSA is a 501(c)(3) non-profit corporation, this option will often be financially preferable to organizing competitions as an individual.
+        CubingUSA offers local organizers the <b>option</b> of running their competition under the auspices of CubingUSA.  Because CubingUSA is a registered 501(c)(3) non-profit corporation, this option may be financially preferable to organizing competitions as an individual. Supported Competitions must follow CubingUSA guidelines to ensure financial accountability.
       </p>
       <h3>Requirements</h3>
       <p>
-        Organizers must submit an application including a budget and other competition details at least <b>60 days in advance</b>.  Income and expenses need to be pre-approved, and generally should be paid or received by CubingUSA.  In order ensure that non-profit laws are followed, there are limitations to what expenses a competition may pay.
+        Organizers must submit an application including a budget and other competition details at least <b>60 days in advance</b>.  Income and expenses need to be pre-approved.  All income goes to CubingUSA, and expenses may be re-imbursed with receipts.  There are limitations on what may be expensed.
       </p>
       <p>
         A <b>Deputy</b> appointed by the CubingUSA Board of Directors must participate in the organization of the competition, and must be present at the competition.  The Deputy is responsible for ensuring all relevant policies are followed, and for reporting any potential issues to the Board of Directors.
       </p>
+      <h3>More Information</h3>
       <p>
-        The full policy is available on our <a href="{{ c.uri_for('documents') }}">Documents</a> page.
+        Please see the <a href="https://docs.google.com/document/d/17OAAiAa1rmxHDEBcFVleGjOR5La39Q-NLIQLROSjzjU/edit?usp=sharing" target="_blank">Supported Competitions FAQ</a> for more details.
```

Comment:
Hmm, it seems a little weird + messy to have two links to this document -- one on google docs and one a pdf that we host.  I see two alternatives...
-have this be a link to the doc hosted in blobstore, which would require knowing which document is the supported competition policy
-do #9 and have the version on the documents page just be a link to the google doc

WDYT?

Reply:
I think the real solution is to rewrite the FAQ and just put it on this page.

The rewrite should make it focus more on practical steps first, THEN go into rationale.  Original FAQ was designed to focus more on introducing the concept.

Happy to just punt on this pull request for now while I work on rewriting this.